hack/jenkins/e2e.sh

```diff
@@ -124,6 +124,21 @@ GCE_PARALLEL_FLAKY_TESTS=(
     "Services.*release.*load\sbalancer"
     )
 
+GCE_SOAK_CONTINUOUS_SKIP=(
```

Comment:
to be consistent with the others, maybe end this with _TESTS?
also, add a comment describing this set, and alphabetize the list.